Meta/Lagom/Tools/CodeGenerators/LibUnicode/GeneratorUtil.h

```diff
@@ -248,6 +250,22 @@ inline ErrorOr<NonnullOwnPtr<Core::Stream::BufferedFile>> open_file(StringView p
     return Core::Stream::BufferedFile::create(move(file));
 }
 
+inline ErrorOr<JsonValue> read_json_file(StringView path)
+{
+    auto file = TRY(open_file(path, Core::Stream::OpenMode::Read));
+
+    StringBuilder builder;
+    Array<u8, 4096> buffer;
+
+    // FIXME: When Core::Stream supports reading an entire file, use that.
+    while (TRY(file->can_read_line())) {
+        auto nread = TRY(file->read(buffer));
+        TRY(builder.try_append(reinterpret_cast<char const*>(buffer.data()), nread));
+    }
```

Comment:
Unless you intend to read an unknown-sized file like stdin, you can just do this:
```c++
auto file_size = TRY(file->size());
auto buffer = TRY(ByteBuffer::create_uninitialized(file_size));
auto nread = TRY(file->read(buffer.bytes()));
VERIFY(nread == file_size);

// Use the contents of the buffer as a StringView
```

Reply:
(though this would proably need a sanity check in the rare case of huge files.)

Reply:
Hmm looks like we'd still need to loop as the call to `read` exits when the stream's buffer is full (i.e. we only read 16384 bytes at a time).